consumer.go

```diff
@@ -533,20 +523,22 @@ func (c *Consumer) maybeWaitForIndexWriteOrAdvance() (didAdvance bool, ok bool,
 	}
 }
 
-func (c *Consumer) hasEnoughIndexForRead() (ok bool, err error) {
-	var indexStat fs.FileInfo
-	indexStat, err = c.indexHandle.Stat()
-	if err != nil {
-		return
-	}
-	var indexPosition int64
-	indexPosition, err = c.indexHandle.Seek(0, io.SeekCurrent)
-	if err != nil {
+func (c *Consumer) tryIndexRead() (ok bool, err error) {
+	// var currentPosition int64
```

Comment:
this may be a yagni thing to leave in here, but figure it's ok in case windows decides to be weird about partial writes